server/seedDB.js

```diff
@@ -52,6 +52,23 @@ const MOCK_USERS = [
     userType: FAMILY,
     userOptions: {
       size: 4,
+      wishlist: [
+        {
+          itemName: 'Backpack',
+          itemQuantity: 1,
+          itemCost: 40,
+        },
+        {
+          itemName: 'Notebook',
+          itemQuantity: 5,
+          itemCost: 3,
+        },
+        {
+          itemName: 'Mechanical Pencil Pack',
+          itemQuantity: 5,
+          itemCost: 8,
```

Comment:
Is this cost per quantity unit? Meaning when we query wishlist.totalCost, we'll get the sum of all (itemQuantity * itemCost) ?

Reply:
Correct, it is cost per unit.  In this case, 5 packs of pencils * $8 = $40 of pencils for the family.

Reply:
Sounds good!

Reply:
So if I understand this correctly, the wishlist APIs will have to do some maths to take itemQuantity * itemCost = itemTotal; and sumOf(itemTotal) = wishlistCost right?

Reply:
Correct again.  Some code would have to be written to do any calculations with the wish list numbers.

We should have a conversation about how to handle situations where not all prices are entered or no prices are entered.  I'll leave that for tonight since it's outside the scope of this PR.